README.md

```diff
@@ -38,6 +38,14 @@ choco install sshs
 pacman -S sshs
 ```
 
+### Alpine Linux
+
+`sshs` is available in Alpine Linux [testing repository](https://pkgs.alpinelinux.org/package/edge/testing/x86_64/sshs).
+
+```shell
+apk add sshs
```

Comment:
```suggestion
apk add sshs --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
```

Isn't the `repository` argument required when using `testing` repository?

The suggestion is an example. I'm not sure if there is an official replicated mirror?

Reply:
It's only required if people doesn't have the testing repository listed in the `/etc/apk/repositories` file, but I would expect Alpine users to know that already.

Reply:
My pleasure, thanks a lot for your wonderful work :)